tests/test_request.py

```diff
@@ -120,6 +120,7 @@ def post(self, request):
 
         return Response(status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
+
```

Comment:
Let's drop these from the pull request.
If they're absolutely required because there's a newer version of the linter, then let's tackle them in a seperate pull request.